applications/nrf_desktop/src/events/config_event.h

```diff
@@ -102,9 +103,8 @@ EVENT_TYPE_DYNDATA_DECLARE(config_event);
 #endif
 
 extern const uint8_t __start_config_channel_modules[];
-extern const uint8_t __stop_config_channel_modules[];
```

Comment:
why stop is removed and start is left?

Reply:
Start is used to get module id in the macro (the module id is generated based on position in the section)